numpy/core/fromnumeric.py

```diff
@@ -63,10 +82,46 @@ def _wrapfunc(obj, method, *args, **kwds):
     # in its class, but its signature is not identical to that
     # of NumPy's. This situation has occurred in the case of
     # a downstream library like 'pandas'.
-    except (AttributeError, TypeError):
+    except (AttributeError, TypeError) as e:
```

Comment:
I don't think it is that bad duplication-wise. I just have the sense that things could be made cleaner. Let me think about it...